tests/benchmark_test.py

```diff
@@ -131,3 +133,131 @@ def test_start_project_from_scratch(mock_collect_user_input):
+
+    # 19 passed, 1 failed - 95% pass rate
+    easy_prompt = (
+        "Fill out the Person class for this data. It needs load_data and __eq__"
+        " functions. The equality functions should compare all attributes and"
+        " recursively compare the parents. A Person should have individual mother"
+        " and father attributes. The load_data function should be a class method of"
+        " Person. The load function will take the data file's path as an argument."
+        " Make sure to pay special attention to the nuances of loading the data. In"
+        " this data file people are stored either as parents, children, or"
+        " standalone. Each one needs to be included. Note that the father is listed"
+        " first in a parents list."
+    )
+    # Removes the line: "Make sure to pay special attention to the nuances of loading the data."
+    # 15 passed, 5 failed - 75% pass rate
+    easy_medium_prompt = (
+        "Fill out the Person class for this data. It needs load_data and __eq__"
+        " functions. The equality functions should compare all attributes and"
+        " recursively compare the parents. A Person should have individual mother"
+        " and father attributes. The load_data function should be a class method of"
+        " Person. The load function will take the data file's path as an argument."
+        " In this data file people are stored either as parents, children, or"
+        " standalone. Each one needs to be included. Note that the father is listed"
+        " first in a parents list."
+    )
+    # Removes the line: "Each one needs to be included."
+    # 8 passed, 12 failed - 40% pass rate
+    medium_prompt = (
+        "Fill out the Person class for this data. It needs load_data and __eq__"
+        " functions. The equality functions should compare all attributes and"
+        " recursively compare the parents. A Person should have individual mother"
+        " and father attributes. The load_data function should be a class method of"
+        " Person. The load function will take the data file's path as an argument."
+        " In this data file people are stored either as parents, children, or"
+        " standalone. Note that the father is listed first in a parents list."
+    )
+    # Removes the Line: "In this data file people are stored either as parents, children, or standalone."
+    # Probably 0% pass rate (I ran it a number of times)
+    hard_prompt = (
+        "Fill out the Person class for this data. It needs load_data and __eq__"
+        " functions. The equality functions should compare all attributes and"
+        " recursively compare the parents. A Person should have individual mother"
+        " and father attributes. The load_data function should be a class method of"
+        " Person. The load function will take the data file's path as an argument."
+        " Note that the father is listed"
+        " first in a parents list."
+    )
+    # makes linter shut up
```

Comment:
You can use # noqa XXX to disable linting warnings on specific lines

Reply:
Thanks, that's better